modules/general/synapse-workspace/main.bicep

```diff
@@ -0,0 +1,286 @@
+@description('The name of the Synapse workspace.')
+param workspaceName string
+
+@description('The location of the Synapse workspace.')
+param location string = resourceGroup().location
+
+@description('The name of the existing Data Lake storage account.')
```

Comment:
Isn't this more like "The name of the storage account that the default data lake file system will be created in"?

I.e. it's not a "Data Lake Storage Account", it's an "Azure Storage Account" which will contain a data lake filesystem?